### PR TITLE
Update Turn-by-Turn demo

### PR DESCRIPTION
**Pull request by erandis-vol**
Prepares the turn-by-turn demo for an update to the latest release. The main focus this time was on simplifying and cleaning things up.

### CODE REVIEW

File: src/TurnByTurn/RoutingSample.Desktop/RoutingSample.Desktop.csproj [hunk 1]
```diff
@@ -151,7 +158,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Esri.ArcGISRuntime.WPF">
-      <Version>100.6.0</Version>
+      <Version>100.7.0</Version>
```
Comment (dotMorten) [inline src/TurnByTurn/RoutingSample.Desktop/RoutingSample.Desktop.csproj:161]: 100.8.0

File: src/TurnByTurn/RoutingSample.Universal/RoutingSample.Universal.csproj [hunk 2]
```diff
@@ -133,7 +140,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Esri.ArcGISRuntime.UWP">
-      <Version>100.6.0</Version>
+      <Version>100.7.0</Version>
```
Comment (dotMorten) [inline src/TurnByTurn/RoutingSample.Universal/RoutingSample.Universal.csproj:143]: 100.8.0

**Comment by twakim** [comment]
@erandis-vol ,  Couple of things when i tried to run this app on iOS
- There was an error trying to access the URL "http://www.arcgis.com/sharing/rest"
  ```The resource could not be loaded because the App Transport Security policy requires the use of a sec…```
- Fixing the first error by changing http to https, I now get a different error:
```
System.UnauthorizedAccessException
  Message=Please set NSLocationWhenInUsageDescription in Info.plist.
```
Is there a setting to enable location on my side that I am missing on iOS?

File: src/TurnByTurn/RoutingSample.Desktop/MainWindow.xaml [hunk 3]
```diff
@@ -44,8 +44,7 @@
         <esri:MapView x:Name="MapView"
                       Map="{Binding Map}"
                       GraphicsOverlays="{Binding RouteGraphicsOverlays}"
-                      Grid.Row="2"
-                      GeoViewTapped="MapView_GeoViewTapped">
```
Comment (twakim) [inline src/TurnByTurn/RoutingSample.Desktop/MainWindow.xaml:48]: Is there a reason why you removed the GeoViewTapped event?  The suer will not be able to click on the map and choose a location to use.

**Comment by twakim** [comment]
@erandis-vol Not sure what is happening on iOS (test device: iPad Pro 3rd Gen)
I do not get the login UI for ArcGIS Online, and neither see the UI that allows me to enter the address to navigate to!

**Comment by twakim** [comment]
WPF and UWP seem to be working fine.

**Comment by twakim** [comment]
@erandis-vol Latest updates seem to be working on iOS.

**Pull request merged**